src/test/resources/org/broadinstitute/hellbender/utils/wdltest/TestWDLTool.wdl

```diff
@@ -0,0 +1,284 @@
+version 1.0
```

Comment:
Expected results files should be clearly labeled as such, either by being put into a directory with "expected" in the name, or by having "expected" in their file names.

Reply:
Done.